XMonad/Actions/DynamicWorkspaceGroups.hs

```diff
@@ -102,21 +107,25 @@ forgetWSGroup :: WSGroupId -> X ()
 forgetWSGroup = XS.modify . withWSG . M.delete
 
 -- | View the workspace group with the given name.
-viewWSGroup :: WSGroupId -> X ()
-viewWSGroup name = do
+viewWSGroup ::  WSGroupId -> X ()
```

Comment:
Why the added space?

Reply:
Because I trained my brain to auto-format each file and not check to format manually... and I can't autoformat files that I didn't write